### ISSUE TITLE
How are Value Types exposed in JS?

### ISSUE THREAD

**Issue by chicoxyzzy** [tschneidereit/proposal-typed-objects#5]
Both overview from this repo and WebAssembly's [GC v1 JS API](https://github.com/WebAssembly/gc/blob/add-js-api/proposals/gc/MVP-JS.md#other-webassembly-value-types) don't mention how Value Types are exposed in JS. In examples they look like Well-known Intrinsic Objects, but they can't be Well-known Intrinsic Objects because they are surely not web compatible. So it's not clear where do `uint8`, `int8`, `any`, etc. come from.

**Comment by tschneidereit** [comment]
True, that part isn't in the explainer yet. That's because we haven't settled on a solution—Typed Objects might end up being exposed as a [builtin module](https://github.com/tc39/proposal-javascript-standard-library), or in a way that's similar to `Math`.
